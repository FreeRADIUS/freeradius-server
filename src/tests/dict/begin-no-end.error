#
#  BEGIN without END
#
BEGIN-VENDOR Cisco
