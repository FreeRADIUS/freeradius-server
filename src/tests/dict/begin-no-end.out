build/tests/dict/begin-no-end.error.txt[3]: BEGIN-VENDOR at src/tests/dict/begin-no-end.error[4] is missing END-VENDOR
Failed parsing dictionary at src/tests/dict/begin-no-end.error[4]
